app-backend/build.sbt

```diff
@@ -693,7 +694,6 @@ lazy val backsplashServer =
 lazy val http4sUtil = Project("http4s-util", file("http4s-util"))
   .dependsOn(db)
   .settings(sharedSettings: _*)
-  .settings(publishSettings)
```

Comment:
`publishSettings` here is implicit since `sharedSettings` has `publishSettings` within it.